charts/kusion/README.md

```diff
@@ -85,8 +91,8 @@ helm install kusion-release kusionstack/kusion \
 --set server.replicas=3 \
 --set mysql.enabled=true \
 --set mysql.database=kusion \
---set kubeconfig.kubeConfigs.kubeconfig0="$KUBECONFIG_CONTENT1" \
---set kubeconfig.kubeConfigs.kubeconfig1="$KUBECONFIG_CONTENT2"
+--set kubeconfig.kubeConfigs.kubeconfig-0="$KUBECONFIG_CONTENT1" \
+--set kubeconfig.kubeConfigs.kubeconfig-1="$KUBECONFIG_CONTENT2"
```

Comment:
The base64 encoding command could be simplified to `base64 -w 0` instead of using `cat` and `tr` for better readability and efficiency.